### PR TITLE
Update constraints for company and student-session

### LINKED ISSUE #190: Handle Delete Company constraints

### PR DESCRIPTION
**Pull request by antongoransson**
We can now properly remove a company, will nilify it's id in student sessions. Closes #190

### CODE REVIEW

File: priv/repo/migrations/20181005115709_update-student-session-constraint.exs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+defmodule :"Elixir.Nexpo.Repo.Migrations.Update-student-session-on-delete" do
```
Comment (VictorWinberg) [inline priv/repo/migrations/20181005115709_update-student-session-constraint.exs]: Dafuq happend here 😸

File: priv/repo/migrations/20181005115709_update-student-session-constraint.exs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-defmodule :"Elixir.Nexpo.Repo.Migrations.Update-student-session-constraint" do
+defmodule Nexpo.Repo.Migrations.UpdateStudentSessionConstraint do
```
Comment (VictorWinberg) [inline priv/repo/migrations/20181005115709_update-student-session-constraint.exs]: Shouldn't it look like this?

Comment (antongoransson) [reply priv/repo/migrations/20181005115709_update-student-session-constraint.exs]: Dont think so https://github.com/careerfairsystems/nexpo/blob/master/priv/repo/migrations/20181001082745_update-student-session-on-delete.exs

Comment (antongoransson) [reply priv/repo/migrations/20181005115709_update-student-session-constraint.exs]: Or that one is wrong also

Comment (VictorWinberg) [reply priv/repo/migrations/20181005115709_update-student-session-constraint.exs]: Lul that looks weird xD

Comment (VictorWinberg) [reply priv/repo/migrations/20181005115709_update-student-session-constraint.exs]: Can't change the past tho

**Pull request merged**